docs/source_files/pixi_tomls/pixi_build_cpp/pixi.toml

```diff
@@ -0,0 +1,32 @@
+[workspace]
+channels = ["https://prefix.dev/conda-forge"]
+name = "python_bindings"
+platforms = ["osx-arm64"]
+preview = ["pixi-build"]                      # (1)!
+
+[dependencies] # (2)!
+python_bindings = { path = "." }
+
+[feature.test.dependencies]
+python = ">=3.12, <3.13"
+
+[feature.test.tasks]
+test = "python -c 'import python_bindings as b; print(b.add(1, 2))'" # (3)!
```

Comment:
Putting that in a separate feature, is a good idea in general, but I think we benefit more by keeping this example as simple as possible. Let's put everything in the `default` feature`

Reply:
Done